OpenSim/Common/DataTable.h

```diff
@@ -341,38 +485,151 @@ class DataTable_ : public AbstractDataTable {
         return _indData;
     }
 
-    /** Get dependent column.                                                 
+    /** Get dependent column at index.
 
-    \throws ColumnIndexOutOfRange If index is out of range.                   */
+    \throws ColumnIndexOutOfRange If index is out of range for number of columns
+                                  in the table.                               */
     VectorView getDependentColumnAtIndex(size_t index) const {
```

Comment:
I'm fine with `getColumnLabels()`, etc., Using a name like `getDependentColumnLabels()` would then make it sound like there should be `getIndependentColumnLabel()`. Also, `DependentColumnLabel` is very long; if we are renaming methods, I'd want to get rid of `Dependent`. I think it's sufficient just to document `getColumnLabels()` saying that it does *not* include the independent column.